### DISCUSSION TITLE
[QUESTION] Backup is not working getting 504 request time out error

### DISCUSSION THREAD

**Discussion by sharanbabumg** [longhorn/longhorn#5799] [Q&A]
## Question
> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.

## Environment

 - Longhorn version: 1.4.1
 - Kubernetes version: 1.25.3
 - Node config
   - OS type and version: redhat linux 8.6
   - CPU per node: 6CPU
   - Memory per node: 32GB
   - Disk type 
   - Network bandwidth and latency between the nodes:
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): VM(on premises) kubernetes cluster(vanilla)

##  Backup is not working getting 504 request time out error. Earlier it was working fine but now backup is completely not working and facing lot of issues. We have deployed longhorn in production also there its working fine as of now.

Only changes we have done is we have added separate disk and partition and default disk disabled and extra disk added is enabled for storage scheduling on longhorn ui.

Can you suggest what could be the issue? Because its very important and critical data is saving on these longhorn volumes.

**Comment by weizhe0422** [comment]
Does the 504 status code appear during UI operations, and does the status code contain error messages?

**Comment by UntouchedWagons** [comment]
Did you solve this? I'm having this issue too. Manual backups and automated backups fail

**Reply by innobead** [reply]
@ChanYiLin Please check this thread as well. Thanks.

@UntouchedWagons can you provide the support bundle? thanks.

**Comment by ChanYiLin** [comment]
Hi @sharanbabumg @UntouchedWagons 
Could you help me to provide the support bundle for us to checkout the logs?
Thanks!

**Comment by UntouchedWagons** [comment]
https://github.com/longhorn/longhorn/discussions/5931#discussioncomment-5939083
